### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by g-th-minh** [gbaranski/quizizz-cheat#43]
![image](https://user-images.githubusercontent.com/92716804/137694966-4443319e-38e7-4997-9770-0d40c4df307f.png)
the code is notworking, the fetching API one. Please fix it, it helps me alot, thank you very much

**Comment by DioOverheaven** [comment]
> ![image](https://user-images.githubusercontent.com/92716804/137694966-4443319e-38e7-4997-9770-0d40c4df307f.png) the code is notworking, the fetching API one. Please fix it, it helps me alot, thank you very much

use the older method it works i just tried it

**Comment by MrCrypticxDev** [comment]
Quizizz just made a change to their endpoint, instead of `/api` in the endpoint just do `/_api` so just copy that and edit where it says `'api'` in the code and that'll work.

**Comment by ChrisVineyard-abc** [comment]
> Quizizz just made a change to their endpoint, instead of `/api` in the endpoint just do `/_api` so just copy that and edit where it says `'api'` in the code and that'll work.

can you show me step by step?

**Comment by JustinYin2** [comment]
```
var a0_0x39f7=['forEach','return','t-componen','body\x20>\x20div','1BdgBEz','pop','filter','v\x20>\x20div\x20>\x20','71irWuHZ','retreive\x20q','>\x20div','opacity','structure','_vm','Could\x20not\x20','ame\x20>\x20div.','querySelec','player','container\x20','next','then','options','roomHash','anski#5119','1UPWQpK','quizId','number','>\x20div.page','length','iterator','log','435216GexlWX','style','error','roomCode','children','uestions\x20l','data','apply','text','.in-quiz\x20>','\x20or\x20','389705VJHjWy','https://qu','__vue__','_data','\x20>\x20div.roo','sent','some','ript\x20creat','label','tor','izz-cheat\x0a','done','806639DCyQdd','n:\x20','push','645485ZlqSKK','map','$store','optionData','ioner-comp','_api/main/g','call','throw','-container','\x20div.scree','anski/quiz','778964IqEKmL','20%','questionID','ops','questions','join','v.options-','json','value','n.screen-g','isArray','retreive\x20r','function','8801SavDRh','trys','437401jcrRsO','playerId','\x20executing','oot\x20object','%c\x20\x0a\x20\x20\x20\x20Sc','ps://githu','answer','actualInde','onent\x20>\x20di','Generator\x20','game','Fail\x20detec','$$state','\x20\x20\x20\x20\x20\x20','Unable\x20to\x20','div\x20>\x20div\x20'];var a0_0x5915=function(_0x3e04e6,_0xd2a1ba){_0x3e04e6=_0x3e04e6-0x146;var _0x39f71e=a0_0x39f7[_0x3e04e6];return _0x39f71e;};(function(_0x1b7f52,_0x6f649d){var _0x27758b=a0_0x5915;while(!![]){try{var _0x4f3955=parseInt(_0x27758b(0x190))*-parseInt(_0x27758b(0x147))+parseInt(_0x27758b(0x157))*-parseInt(_0x27758b(0x192))+parseInt(_0x27758b(0x175))+parseInt(_0x27758b(0x15e))+-parseInt(_0x27758b(0x183))+parseInt(_0x27758b(0x178))*parseInt(_0x27758b(0x1a6))+parseInt(_0x27758b(0x169));if(_0x4f3955===_0x6f649d)break;else _0x1b7f52['push'](_0x1b7f52['shift']());}catch(_0x26c328){_0x1b7f52['push'](_0x1b7f52['shift']());}}}(a0_0x39f7,0x6a661),(()=>{'use strict';var _0xfde10d,_0x1abb38,_0x34fd8f=function(_0x209968){var _0x8719df=a0_0x5915;_0x209968[_0x8719df(0x15f)][_0x8719df(0x14a)]=_0x8719df(0x184);},_0x19bb79=function(_0x516685){var _0x188d5e=a0_0x5915,_0x49a9ac=function(){var _0xc6f452=a0_0x5915,_0x3d1b6d=document[_0xc6f452(0x14f)+'tor'](_0xc6f452(0x1a5)+_0xc6f452(0x16d)+_0xc6f452(0x1a4)+'t\x20>\x20div\x20>\x20'+'div\x20>\x20div\x20'+_0xc6f452(0x15a)+_0xc6f452(0x180)+_0xc6f452(0x167)+_0xc6f452(0x181)+_0xc6f452(0x18c)+_0xc6f452(0x14e)+'transition'+'er.transit'+_0xc6f452(0x17c)+_0xc6f452(0x19a)+_0xc6f452(0x146)+_0xc6f452(0x1a1)+'>\x20div\x20>\x20di'+_0xc6f452(0x189)+_0xc6f452(0x151)+_0xc6f452(0x149));if(!_0x3d1b6d)throw new Error(_0xc6f452(0x1a0)+_0xc6f452(0x148)+_0xc6f452(0x163)+'ist\x20elemen'+'t');return _0x3d1b6d;}(),_0xf5ab78=Array['prototype']['slice'][_0x188d5e(0x17e)](_0x49a9ac[_0x188d5e(0x162)]);if(Array[_0x188d5e(0x18d)](_0x516685[_0x188d5e(0x14b)][_0x188d5e(0x198)])&&_0x516685[_0x188d5e(0x14b)][_0x188d5e(0x198)][_0x188d5e(0x15b)]<0x1&&_0x516685[_0x188d5e(0x14b)][_0x188d5e(0x154)]){var _0x17f22c=_0x516685['structure'][_0x188d5e(0x154)][_0x188d5e(0x179)](function(_0xe7248){var _0x183afd=_0x188d5e;return _0xe7248[_0x183afd(0x166)];})[_0x188d5e(0x188)](_0x188d5e(0x168));alert(_0x17f22c);}else _0xf5ab78[_0x188d5e(0x1a8)](function(_0x3a601e){var _0x5a79bc=_0x188d5e;return Array[_0x5a79bc(0x18d)](_0x516685[_0x5a79bc(0x14b)][_0x5a79bc(0x198)])&&_0x516685['structure']['answer'][_0x5a79bc(0x15b)]>0x0?!_0x516685[_0x5a79bc(0x14b)]['answer'][_0x5a79bc(0x16f)](function(_0x3879d5){var _0x3e872a=_0x5a79bc;return _0x3a601e[_0x3e872a(0x16b)][_0x3e872a(0x17b)][_0x3e872a(0x199)+'x']===_0x3879d5;}):_0x5a79bc(0x159)==typeof _0x516685[_0x5a79bc(0x14b)][_0x5a79bc(0x198)]?_0x3a601e[_0x5a79bc(0x16b)][_0x5a79bc(0x17b)][_0x5a79bc(0x199)+'x']!==_0x516685['structure'][_0x5a79bc(0x198)]:void console[_0x5a79bc(0x160)](_0x5a79bc(0x19d)+'ting\x20type\x20'+'of\x20questio'+_0x5a79bc(0x176),_0x516685);})[_0x188d5e(0x1a2)](_0x34fd8f);},_0x1fc497=function(){var _0x208c44=a0_0x5915,_0x1503b4=document[_0x208c44(0x14f)+_0x208c44(0x172)](_0x208c44(0x1a5));if(!_0x1503b4)throw new Error('Could\x20not\x20'+_0x208c44(0x18e)+_0x208c44(0x195));return _0x1503b4[_0x208c44(0x16b)]['$store'][_0x208c44(0x14c)][_0x208c44(0x16c)][_0x208c44(0x19e)][_0x208c44(0x19c)][_0x208c44(0x164)][_0x208c44(0x155)];};_0x1abb38=function(){var _0x3d9974,_0x167e1f;return function(_0x3a8673,_0x4e46b9){var _0x742dad=a0_0x5915,_0x1c57a5,_0x480105,_0xebed39,_0x237930,_0x3c3a65={'label':0x0,'sent':function(){if(0x1&_0xebed39[0x0])throw _0xebed39[0x1];return _0xebed39[0x1];},'trys':[],'ops':[]};return _0x237930={'next':_0x24e050(0x0),'throw':_0x24e050(0x1),'return':_0x24e050(0x2)},_0x742dad(0x18f)==typeof Symbol&&(_0x237930[Symbol[_0x742dad(0x15c)]]=function(){return this;}),_0x237930;function _0x24e050(_0x2309cd){return function(_0x39d9d0){return function(_0xe78ab6){var _0x22fec1=a0_0x5915;if(_0x1c57a5)throw new TypeError(_0x22fec1(0x19b)+'is\x20already'+_0x22fec1(0x194)+'.');for(;_0x3c3a65;)try{if(_0x1c57a5=0x1,_0x480105&&(_0xebed39=0x2&_0xe78ab6[0x0]?_0x480105['return']:_0xe78ab6[0x0]?_0x480105[_0x22fec1(0x17f)]||((_0xebed39=_0x480105[_0x22fec1(0x1a3)])&&_0xebed39[_0x22fec1(0x17e)](_0x480105),0x0):_0x480105[_0x22fec1(0x152)])&&!(_0xebed39=_0xebed39[_0x22fec1(0x17e)](_0x480105,_0xe78ab6[0x1]))['done'])return _0xebed39;switch(_0x480105=0x0,_0xebed39&&(_0xe78ab6=[0x2&_0xe78ab6[0x0],_0xebed39[_0x22fec1(0x18b)]]),_0xe78ab6[0x0]){case 0x0:case 0x1:_0xebed39=_0xe78ab6;break;case 0x4:return _0x3c3a65[_0x22fec1(0x171)]++,{'value':_0xe78ab6[0x1],'done':!0x1};case 0x5:_0x3c3a65[_0x22fec1(0x171)]++,_0x480105=_0xe78ab6[0x1],_0xe78ab6=[0x0];continue;case 0x7:_0xe78ab6=_0x3c3a65['ops'][_0x22fec1(0x1a7)](),_0x3c3a65['trys']['pop']();continue;default:if(!((_0xebed39=(_0xebed39=_0x3c3a65[_0x22fec1(0x191)])[_0x22fec1(0x15b)]>0x0&&_0xebed39[_0xebed39[_0x22fec1(0x15b)]-0x1])||0x6!==_0xe78ab6[0x0]&&0x2!==_0xe78ab6[0x0])){_0x3c3a65=0x0;continue;}if(0x3===_0xe78ab6[0x0]&&(!_0xebed39||_0xe78ab6[0x1]>_0xebed39[0x0]&&_0xe78ab6[0x1]<_0xebed39[0x3])){_0x3c3a65[_0x22fec1(0x171)]=_0xe78ab6[0x1];break;}if(0x6===_0xe78ab6[0x0]&&_0x3c3a65[_0x22fec1(0x171)]<_0xebed39[0x1]){_0x3c3a65[_0x22fec1(0x171)]=_0xebed39[0x1],_0xebed39=_0xe78ab6;break;}if(_0xebed39&&_0x3c3a65['label']<_0xebed39[0x2]){_0x3c3a65[_0x22fec1(0x171)]=_0xebed39[0x2],_0x3c3a65[_0x22fec1(0x186)][_0x22fec1(0x177)](_0xe78ab6);break;}_0xebed39[0x2]&&_0x3c3a65[_0x22fec1(0x186)][_0x22fec1(0x1a7)](),_0x3c3a65[_0x22fec1(0x191)][_0x22fec1(0x1a7)]();continue;}_0xe78ab6=_0x4e46b9['call'](_0x3a8673,_0x3c3a65);}catch(_0x1ef155){_0xe78ab6=[0x6,_0x1ef155],_0x480105=0x0;}finally{_0x1c57a5=_0xebed39=0x0;}if(0x5&_0xe78ab6[0x0])throw _0xe78ab6[0x1];return{'value':_0xe78ab6[0x0]?_0xe78ab6[0x1]:void 0x0,'done':!0x0};}([_0x2309cd,_0x39d9d0]);};}}(this,function(_0x4073d8){var _0x446c0c=a0_0x5915;switch(_0x4073d8['label']){case 0x0:return console[_0x446c0c(0x15d)](_0x446c0c(0x196)+_0x446c0c(0x170)+'ed\x20by\x20gbar'+_0x446c0c(0x156)+'!\x20\x0a\x20\x20\x20\x20htt'+_0x446c0c(0x197)+'b.com/gbar'+_0x446c0c(0x182)+_0x446c0c(0x173)+_0x446c0c(0x19f),'color:\x20red'+';'),[0x4,fetch(_0x446c0c(0x16a)+'izizz.com/'+_0x446c0c(0x17d)+'ame/'+_0x1fc497())];case 0x1:return[0x4,_0x4073d8['sent']()[_0x446c0c(0x18a)]()];case 0x2:return _0x3d9974=_0x4073d8[_0x446c0c(0x16e)](),_0x167e1f=void 0x0,setInterval(function(){var _0x27f1e8=_0x446c0c,_0x3944ee=function(){var _0x280cbb=a0_0x5915,_0x1843b6=document[_0x280cbb(0x14f)+_0x280cbb(0x172)]('body\x20>\x20div');if(!_0x1843b6)throw new Error(_0x280cbb(0x14d)+_0x280cbb(0x18e)+_0x280cbb(0x195));var _0x4239f2=_0x1843b6[_0x280cbb(0x16b)];return{'roomHash':_0x4239f2['$store'][_0x280cbb(0x14c)]['_data'][_0x280cbb(0x19e)]['game']['data'][_0x280cbb(0x155)],'playerId':_0x4239f2['$store'][_0x280cbb(0x14c)][_0x280cbb(0x16c)][_0x280cbb(0x19e)][_0x280cbb(0x19c)][_0x280cbb(0x150)][_0x280cbb(0x193)],'quizID':_0x4239f2['$store'][_0x280cbb(0x14c)][_0x280cbb(0x16c)][_0x280cbb(0x19e)][_0x280cbb(0x19c)][_0x280cbb(0x164)][_0x280cbb(0x158)],'roomCode':_0x4239f2[_0x280cbb(0x17a)][_0x280cbb(0x14c)][_0x280cbb(0x16c)][_0x280cbb(0x19e)]['game'][_0x280cbb(0x164)][_0x280cbb(0x161)],'questionID':_0x4239f2[_0x280cbb(0x17a)][_0x280cbb(0x14c)][_0x280cbb(0x16c)][_0x280cbb(0x19e)][_0x280cbb(0x19c)]['questions']['currentId']};}();if(_0x3944ee['questionID']!==_0x167e1f)for(var _0x5ecec4=0x0,_0x4a6595=_0x3d9974[_0x27f1e8(0x164)][_0x27f1e8(0x187)];_0x5ecec4<_0x4a6595[_0x27f1e8(0x15b)];_0x5ecec4++){var _0x1ad985=_0x4a6595[_0x5ecec4];_0x3944ee[_0x27f1e8(0x185)]===_0x1ad985['_id']&&(console[_0x27f1e8(0x15d)]({'q':_0x1ad985}),_0x19bb79(_0x1ad985),_0x167e1f=_0x3944ee[_0x27f1e8(0x185)]);}},0x1f4),[0x2];}});},new((_0xfde10d=void 0x0)||(_0xfde10d=Promise))(function(_0xd9cb26,_0x17340b){var _0x341207=a0_0x5915;function _0x1598fe(_0x81626){var _0xde43f1=a0_0x5915;try{_0x241e48(_0x1abb38[_0xde43f1(0x152)](_0x81626));}catch(_0x1b1cb7){_0x17340b(_0x1b1cb7);}}function _0x25a0dc(_0x229005){var _0x2840d4=a0_0x5915;try{_0x241e48(_0x1abb38[_0x2840d4(0x17f)](_0x229005));}catch(_0x5966c0){_0x17340b(_0x5966c0);}}function _0x241e48(_0x5a3600){var _0x40f049=a0_0x5915,_0x3ff97d;_0x5a3600[_0x40f049(0x174)]?_0xd9cb26(_0x5a3600[_0x40f049(0x18b)]):(_0x3ff97d=_0x5a3600[_0x40f049(0x18b)],_0x3ff97d instanceof _0xfde10d?_0x3ff97d:new _0xfde10d(function(_0x15757f){_0x15757f(_0x3ff97d);}))[_0x40f049(0x153)](_0x1598fe,_0x25a0dc);}_0x241e48((_0x1abb38=_0x1abb38[_0x341207(0x165)](void 0x0,[]))[_0x341207(0x152)]());});})());
```

**Comment by JustinYin2** [comment]
try lmao i just copy the entire dist file and change api to _api

**Comment by ChrisVineyard-abc** [comment]
> try lmao i just copy the entire dist file and change api to _api

it's working, :x thanks mate

**Comment by MrCrypticxDev** [comment]
You are welcome!

**Comment by gbaranski** [comment]
closed by #44
